x-pack/plugins/lens/public/app_plugin/mounter.tsx

```diff
@@ -92,7 +88,12 @@ export async function mountApp(
   const renderEditor = (routeProps: RouteComponentProps<{ id?: string }>) => {
     trackUiEvent('loaded');
     const urlParams = parse(routeProps.location.search) as Record<string, string>;
-    const originatingApp = urlParams.embeddableOriginatingApp;
+    const originatingAppFromUrl = urlParams.embeddableOriginatingApp;
+    if (urlParams.embeddableOriginatingApp) {
+      setTimeout(() => {
```

Comment:
Why do we need to get async here?

Reply:
This used to be needed due to the url change and the render method creating an infinite loop. This is no longer necessary, so it's been removed!